specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceHealthMetadata.json

```diff
@@ -0,0 +1,373 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-03-01",
+    "title": "ResourceHealthMetadata API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/resourceHealthMetadata": {
+      "get": {
+        "tags": [
+          "ResourceHealthMetadata"
+        ],
+        "summary": "List all ResourceHealthMetadata for all sites in the subscription.",
+        "description": "List all ResourceHealthMetadata for all sites in the subscription.",
+        "operationId": "ResourceHealthMetadata_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceHealthMetadataCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List ResourceHealthMetadata for a subscription": {
+            "$ref": "./examples/ListResourceHealthMetadataBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/resourceHealthMetadata": {
+      "get": {
+        "tags": [
+          "ResourceHealthMetadata"
+        ],
+        "summary": "List all ResourceHealthMetadata for all sites in the resource group in the subscription.",
+        "description": "List all ResourceHealthMetadata for all sites in the resource group in the subscription.",
+        "operationId": "ResourceHealthMetadata_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Add "default" response to all APIs to capture error cases. Schema complies with the ARM RPC error contract. example from Batch RP swagger.

Reply:
@ravbhatnagar This looks to be a newer requirement for swagger files. Since we (Microsoft.Web) auto-generate our swagger files from code, is it okay to not have the "default" response for this file while we work on adding support for this in the auto-generation?

Reply:
Fixed up the responses to have a default error response